examples/model_loader.py

```diff
@@ -0,0 +1,94 @@
+from yggdrasil.examples import yamls as ex_yamls
```

Comment:
We should not be importing from `examples`

Reply:
This was originally used for a photosynthesis parsing example, but since it's broken there's no need to use it. removed as told